.golangci.yaml

```diff
@@ -4,11 +4,29 @@ linters:
   enable:
     - unused
     - govet
+    - importas
     # More linters can be added, see supported ones: https://golangci-lint.run/usage/linters/
   exclusions:
     presets:
       - common-false-positives
       - std-error-handling
+  settings:
+    importas:
+      no-unaliased: true
+      alias:
+        # # e.g. k8s.io/api/core/v1 -> corev1
+        - pkg: k8s.io/api/(\w+)/(v[\w\d]+)
+          alias: ${1}${2}
+        # e.g. github.com/scylladb/scylla-operator/pkg/api/scylla/v1alpha1 -> scyllav1alpha1
+        - pkg: github.com/scylladb/scylla-operator/pkg/api/(\w+)/(v\d+[\w\d]*)
+          alias: ${1}${2}
+        # e.g. k8s.io/apimachinery/pkg/util/errors -> k8serrors
+        - pkg: k8s.io/apimachinery/pkg/util/(\w+)
+          alias: k8s${1}
+        - pkg: k8s.io/apimachinery/pkg/apis/meta/v1
```

Comment:
nit: `apimachinerymetav1validation "k8s.io/apimachinery/pkg/apis/meta/v1/validation"` might also be worth adding to set up a pattern

Reply:
:+1: Added with the `metav1validation` alias.